packages/@uppy/provider-views/src/index.js

```diff
@@ -458,8 +458,8 @@ module.exports = class ProviderView {
 
     const patterns = Array.isArray(allowedOrigin) ? allowedOrigin.map(getRegex) : [getRegex(allowedOrigin)]
     return patterns
-      .filter((pattern) => pattern !== null)
-      .some((pattern) => pattern.test(origin))
+      .filter((pattern) => pattern != null) // loose comparison to catch undefined
```

Comment:
I wonder if it’s better to explicitly check for `undefined` for readability. But with your comment this works, just thinking out loud.